__tests__/lib/pg/service.test.js

```diff
@@ -138,10 +138,36 @@ describe('OData to Postgres dialect', () => {
     })
   })
 
+  describe('odata: PUT -> sql: UPDATE', () => {
+    beforeEach(async () => {
+      await deploy(this._model, {}).to(this._dbProperties)
+    })
+
+    test('odata: entityset Beers -> sql: update beers', async () => {
+      const response = await request
+        .put('/beershop/Beers/9e1704e3-6fd0-4a5d-bfb1-13ac47f7976b')
+        .send({
+          name: 'Changed name',
+          ibu: 10,
+        })
+        .set('content-type', 'application/json;charset=UTF-8;IEEE754Compatible=true')
+      expect(response.status).toStrictEqual(200)
+
+      const getResponse = await request.get('/beershop/Beers/9e1704e3-6fd0-4a5d-bfb1-13ac47f7976b').send()
+      expect(getResponse.body).toEqual(expect.objectContaining({ name: 'Changed name', ibu: 10 }))
+    })
+  })
+
   describe('odata: DELETE -> sql: DELETE', () => {
     test('odata delete ', async () => {
       const response = await request.delete(`/beershop/Beers/9e1704e3-6fd0-4a5d-bfb1-13ac47f7976b`).send()
       expect(response.status).toStrictEqual(204)
+
+      // REVISIT: Works and test does not fail but console contains express error (404). How to solve this?
```

Comment:
don't really understand the REST-type query here...probably left-over to be deleted? ;)

Reply:
And I don't understand your comment. ;-)
What do you mean by REST-type query?

Reply:
the commented out part with "REVISIT"

Reply:
Still confused. :-)
I wanted to check if the record is really gone in the db. But when you fire the next GET request, the odata server responds with a 404 and throws an exception that is visible in the terminal output, but does not affect the test results.

Reply:
ah, ok, my bad. now I get it (coffee helped!). let me try out a thingie or two and get back to you on why the subsequent GET doesn't resolve to the desired 404.

Reply:
this seems to work fine for me:
```js
  test('odata: delete single beer -> sql: delete record', async () => {
      const guid = '9e1704e3-6fd0-4a5d-bfb1-13ac47f7976b'
      const response = await request.delete(`/beershop/Beers(${guid})`)
      expect(response.status).toStrictEqual(204)

      // make sure the deleted beer doesn't exist anymore
      const subsequentResponse = await request.get(`/beershop/Beers(${guid})`)
      expect(subsequentResponse.status).toStrictEqual(404)
    })
```
can you double-check please?

Reply:
This pops up on my machine. The test itself does not break, but I get an ugly terminal output.
If that's not happening on your machine or if it is fine, that let's merge this.

![image](https://user-images.githubusercontent.com/9880/94537529-5a8a2780-0243-11eb-9b3e-95a7585a1204.png)

Reply:
jep, same here - and that's fine for me given that it's the @sap/cds odata server logging the 404